docs/install.md

```diff
@@ -0,0 +1,88 @@
+# Installation
+
+This package provides both a C++ evaluator with python bindings and
+a set of python tools to convert, validate, and inspect files with corrections
+It is installed as a python package (see e.g.
+[this tutorial](https://packaging.python.org/tutorials/installing-packages/)
+for an introduction), using [scikit-build](https://scikit-build.readthedocs.io/)
+and thus [CMake](https://cmake.org/), for the C++ components.
+Builds have been tested in Windows, OS X, and Linux, and the python bindings
+can be compiled against both python2 (with limited functionality) and python3,
+as well as from within a CMSSW environment.
+Note that CMSSW 11_2_X and above has ROOT accessible from python 3.
+
+## Inside CMSSW
+
+For CMSSW releases from the `11_3_X` branch on, correctionlib is included
+as the `py3-correctionlib` tool, so no installation is needed.
```

Comment:
In https://github.com/cms-sw/cmsdist/pull/7301 correctionlib was updated to v2. As very few corrections were produced in v1 schema, we might want to add an instruction to check the version, which could be done by testing the result of `correction config --version`, and if that command fails then definitely the version is too old. Right now I think its the case that only 12_1_X branch has it (e.g. `CMSSW_12_1_0_pre4`). I need to look into the possibility of backporting